tests/twister2_plugin_quarantine_test.py

```diff
@@ -1,42 +1,31 @@
 
-import json
 import textwrap
 from pathlib import Path
 
-
-def _get_test_scenario(tests, test_name):
-    matched_scenarios = [scenario for scenario in tests if scenario['test_name'] == test_name]
-    if not matched_scenarios:
-        return None
-    return matched_scenarios[0]
+import pytest
 
 
 def test_if_pytest_use_quarantine_file(pytester, resources) -> None:
     """
-    Run tests with quarantine list. Verify in test output json file
-    if quarantine comment is added to proper test configuration
+    Run tests with quarantine list. Verify test output
     """
     pytester.copy_example(str(resources))
     quarantine_file: Path = resources / 'quarantine' / 'helloworld_native.yml'
-    output_testplan: Path = pytester.path / 'testplan.json'
-    pytester.runpytest(
+    result = pytester.runpytest(
         f'--zephyr-base={str(pytester.path)}',
         '--platform=native_posix',
         '--platform=qemu_cortex_m3',
         f'--quarantine-list={quarantine_file}',
-        f'--testplan-json={output_testplan}',
         '--co'
     )
-    with open(output_testplan) as f:
-        json_data = json.load(f)
-
-    hello_qemu = _get_test_scenario(json_data['tests'], 'sample.basic.helloworld[qemu_cortex_m3]')
-    assert hello_qemu
-    assert not hello_qemu['quarantine']
+    result.stdout.re_match_lines_random([
+        r'.*sample.basic.helloworld\[qemu_cortex_m3\]*'
+    ])
+    result.stdout.no_re_match_line(r'.*sample.basic.helloworld\[native_posix\]*')
 
-    hello_native = _get_test_scenario(json_data['tests'], 'sample.basic.helloworld[native_posix]')
-    assert hello_native
-    assert 'link.to.issue' in hello_native['quarantine']
+    # check if quarantine comment is found in output log
+    with open(Path('twister-out/testcases_creation.log')) as f:
```

Comment:
Is it work without providing full path (pytester.path) to log file?

Reply:
yes, it works